pkg/custom_models/json.go

```diff
@@ -0,0 +1,15 @@
+package custom_models
+
+import (
+	"bytes"
+	"encoding/json"
+)
+
+type JSON map[string]interface{}
+
+// Basic Unmarshal. Do not UseNumber (unmarshal to float64 over json.Number)
+func (j *JSON) UnmarshalJSON(data []byte) error {
+	type j2 JSON
+	dec := json.NewDecoder(bytes.NewReader(data))
+	return dec.Decode((*j2)(j))
```

Comment:
Why using a `json.Decoder` and not calling `json.Unmarshal` directly?

Reply:
I believe the issue is that `json.Unmarshal` has the default behavior of unmarshalling as `json.Number`. I will test